### ISSUE TITLE
DELETE request should return the id of the deleted record

### ISSUE THREAD

**Issue by mikemfleming** [typicode/json-server#671]
Or at least an option for that to happen. I don't know that it's an official RESTful practice but it's one that I usually employ. Thoughts?

**Comment by RobinLebhar** [comment]
Yeah delete should return the id of the complete entity, it's ofter use, to display which entity has been deleted, or if you want to easily undo it.
Moreover it indicate if it worked or not.
That is a must have, clearly !

**Comment by jacobwarduk** [comment]
I'd have to agree, and would be more than happy to pick this up if it is approved as a new feature to implement.
